resources/data/layers/default.json

```diff
@@ -145,64 +153,57 @@
     },
     {
       "layerType": "switchlayer",
-      "layerKey": "LIGHT_UNIT_SWITCH_LAYER",
+      "layerKey": "WATERBODY_SWITCH_LAYER",
       "visibility": false,
       "layers": [
         {
           "layerType": "wms",
-          "text": "Light Unit Switch Layer (far)",
-          "layerKey": "LIGHT_UNIT_SWITCH_LAYER_FAR",
+          "text": "Waterbody Switch Layer (far)",
+          "layerKey": "WATERBODY_SWITCH_LAYER_FAR",
           "serverOptions": {
-            "layers": "LightUnit",
-            "version": "1.3.0"
+            "layers": "waterbodies"
           },
           "openLayers": {
             "singleTile": false
           },
           "styles": [
-            "Unit Type",
-            "Owner"
+            "Waterbodies",
+            "Type"
           ],
           "labelClassName": "labels"
         },
         {
           "layerType": "wfs",
-          "text": "Light Unit Switch Layer (close)",
-          "layerKey": "LIGHT_UNIT_SWITCH_LAYER_CLOSE",
-          "url": "https://plmonaghandev.compass.ie/mapserver/?",
+          "text": "Waterbody Switch Layer (close)",
+          "layerKey": "WATERBODY_SWITCH_LAYER_CLOSE",
           "geometryProperty": "msGeometry",
-          "featureType": "LightUnit",
-          "sldUrl": "resources/data/styling/LightUnit_Unit_Type.xml",
-          "openLayers": {},
-          "serverOptions": {
-            "outputFormat": "geojson"
-          },
+          "featureType": "waterbodies",
           "noCluster": true,
-          "stylesBaseUrl": "https://plmonaghandev.compass.ie/resources/data/styling/",
           "styles": [
-            "LightUnit_Type.xml",
-            "LightUnit_Owner.xml"
+            "Waterbodies_Default.xml",
+            "Waterbodies_Type.xml"
           ],
-          "stylesForceNumericFilterVals": true
+          "tooltipsConfig": [
+            {
+              "alias": "Id",
+              "property": "osm_id"
+            },
+            {
+              "alias": "Code",
+              "property": "code"
+            },
+            {
+              "alias": "Class",
+              "property": "fclass"
+            },
+            {
+              "alias": "Name",
+              "property": "name"
+            }
+          ]
         }
       ]
     },
-    {
-      "layerType": "wms",
-      "text": "Works_Export (needs proxy)",
-      "layerKey": "WORKS_EXPORT_WMS_PROXY",
-      "url": "mapserver/",
-      "timeitem": "TIME",
-      "dateFormat": "Y",
-      "serverOptions": {
-        "layers": "Works_Type",
-        "version": "1.1.1"
-      },
-      "openLayers": {
-        "singleTile": true,
-        "visibility": false
-      }
-    },
     {
       "layerType": "wfs",
       "text": "Test time WFS (needs proxy)",
```

Comment:
Do we need this anymore?

Reply:
@chrismayer - thanks for the feedback. I've added in the vtwms layer type and removed the time layer as suggested.